rpc/documentation/api.md

```diff
@@ -1476,3 +1478,43 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+#### `StartTicketPurchase`
+
+The `StartTicketPurchase` method enables ticket purchases for stake mining in the wallet.
+
+**Request:** `StartTicketPurchaseRequest`
+- `passphrase`: The private passphrase to unlock the wallet.
+- `account_name`: The name of the account being queried.
+- `balance_to_maintain`: The amount of funds to leave in wallet after purchasing tickets.
```

Comment:
More accurate to call this the minimum amount of regular funds to never dip below.  Right now it sounds like we buy tickets to reach this exact value, which can't be done.